pydicer/convert/convert_pt.py

```diff
@@ -0,0 +1,311 @@
+import warnings
+from datetime import time, datetime
+import numpy as np
+import SimpleITK as sitk
+import pydicom as pdcm
+from pydicom.tag import Tag
+
+
+def convert_dicom_to_nifty_pt(

```

Comment:
For the modality="PT" parameter, since this function is for converting PET images only, is it necessary to have this parameter at all? 

Also, is the datatype of the image necessary to pass in this case? Can we assume that the function will work for different floating point numbers? Maybe we should just stick to 32 bit floats only and not allow the user to interface with this...

Reply:
I agree with @dalmouiee here. In addition, I also think we can remove the sitk_writer? We can just use sitk.WriteImage directly.